src/Modules/Button/includes/frontend.php

```diff
@@ -6,14 +6,14 @@
  */
 
 use Itineris\SageFLBuilder\AbstractHelper;
-use function App\sage;
+use function Roots\app;
 
 /** @var AbstractHelper $helper */
-$helper = sage(AbstractHelper::class);
+$helper = app(AbstractHelper::class);
```

Comment:
Redundant? Maybe try making it a blade file.

Reply:
@codepuncher We will convert all these files in blade a bit later, I need to work now on BOHS BB components and I need sage-flbuilder.